addons/website/views/snippets/s_image.xml

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
+<odoo>
+
+<template id="s_image" name="Image">
+    <img class="img img-fluid o_we_custom_image o_snippet_drop_in_only" src="/web/image/website.s_text_image_default_image" alt=""
+        loading="lazy" style="min-height: 1px;" />
```

Comment:
Removing this, it's useless. Loading="lazy" is automatically added on page rendering, not need for it during edition.